src/harness/testcases/WINNF_FT_S_FPR_testcase.py

```diff
@@ -0,0 +1,154 @@
+    gwbl['record']['deploymentParam'][0]['installationParam']['latitude'] = \
+        fss['deploymentParam'][0]['installationParam']['latitude'] + 0.1
+    gwbl['record']['deploymentParam'][0]['installationParam']['longitude'] = \
+        fss['deploymentParam'][0]['installationParam']['longitude']
+
+    # Create info for grants
+    grant_1 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_1['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
+    grant_1['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
+    grant_2 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_2['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
+    grant_2['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
+    grant_3 = json.load(open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_3['operationParam']['operationFrequencyRange']['lowFrequency'] = 3650000000
+    grant_3['operationParam']['operationFrequencyRange']['highFrequency'] = 3660000000
+
+    # Creating conditionals for Cat B device
+    self.assertEqual(device_1['cbsdCategory'], 'B')
+    conditionalParameters = {
+        'cbsdCategory': device_1['cbsdCategory'],
+        'fccId': device_1['fccId'],
+        'cbsdSerialNumber': device_1['cbsdSerialNumber'],
+        'airInterface': device_1['airInterface'],
+        'installationParam': device_1['installationParam'],
+        'measCapability': device_1['measCapability']
+    }
+    del device_1['cbsdCategory']
+    del device_1['airInterface']
+    del device_1['installationParam']
+    del device_1['measCapability']
+
+    # Create the actual config.
+    conditionals = [conditionalParameters]
+    config = {
+      'registrationRequests': [device_1, device_2, device_3],
+      'conditionalRegistrationData': conditionals,
+      'grantRequests': [grant_1, grant_2, grant_3],
+      'gwblRecord': gwbl,
+      'fssRecord': fss
+    }
+    writeConfig(filename, config)
+
+  @configurable_testcase(generate_FPR_5_default_config)
+  def test_WINNF_FT_S_FPR_5(self, config_filename):
+    """[Configurable] Grant Requests from one or more CBSDs Inside an FSS-GWBL Exclusion Zone."""
+
+    config = loadConfig(config_filename)
+        
+    # Light checking of the config file
+    self.assertEqual(len(config['registrationRequests']), len(config['grantRequests']))
+
+    # Load the FSS
+    self._sas_admin.InjectFss({'record': config['fssRecord']})
+
+    # Load the GWBL
+    self._sas_admin.InjectWisp(config['gwblRecord'])
+
+    # Trigger CPAS activity
+    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete() 
+
+    # Register N > 0 CBSDs
+    # Check registration response
+    # The assertRegistered function does the Inject FCC ID and user ID for the registration requests
+    if ('conditionalRegistrationData' in config) and (config['conditionalRegistrationData']):
```

Comment:
We can just have: 

if ('conditionalRegistrationData' in config):
  cbsd_ids = self.assertRegistered(config['registrationRequests'],config['conditionalRegistrationData'])